README.md

```diff
@@ -46,6 +47,123 @@ Possible output:
wNS_19ostreambuf_iteratorIwNS_11char_traitsIwEEEEEE\x00NSt3__29__num_putIwEE\x00%H:%M:%S\x00%m/%d/%y\x00%I:%M:%S %p\x00%a %b %d %H:%M:%S %Y\x00AM\x00PM\x00January\x00February\x00March\x00April\x00May\x00June\x00July\x00August\x00September\x00October\x00November\x00December\x00Jan\x00Feb\x00Mar\x00Apr\x00Jun\x00Jul\x00Aug\x00Sep\x00Oct\x00Nov\x00Dec\x00Sunday\x00Monday\x00Tuesday\x00Wednesday\x00Thursday\x00Friday\x00Saturday\x00Sun\x00Mon\x00Tue\x00Wed\x00Thu\x00Fri\x00Sat\x00%m/%d/%y%Y-%m-%d%I:%M:%S %p%H:%M%H:%M:%S%H:%M:%SNSt3__28time_getIcNS_19istreambuf_iteratorIcNS_11char_traitsIcEEEEEE\x00NSt3__220__time_get_c_storageIcEE\x00NSt3__29time_baseE\x00NSt3__28time_getIwNS_19istreambuf_iteratorIwNS_11char_traitsIwEEEEEE\x00NSt3__220__time_get_c_storageIwEE\x00NSt3__28time_putIcNS_19ostreambuf_iteratorIcNS_11char_traitsIcEEEEEE\x00NSt3__210__time_putE\x00locale not supported\x00NSt3__28time_putIwNS_19ostreambuf_iteratorIwNS_11char_traitsIwEEEEEE\x00NSt3__210moneypunctIcLb0EEE\x00NSt3__210money_baseE\x00NSt3__210moneypunctIcLb1EEE\x00NSt3__210moneypunctIwLb0EEE\x00NSt3__210moneypunctIwLb1EEE\x000123456789\x00%Lf\x00money_get error\x00NSt3__29money_getIcNS_19istreambuf_iteratorIcNS_11char_traitsIcEEEEEE\x00NSt3__211__money_getIcEE\x000123456789\x00NSt3__29money_getIwNS_19istreambuf_iteratorIwNS_11char_traitsIwEEEEEE\x00NSt3__211__money_getIwEE\x00%.0Lf\x00NSt3__29money_putIcNS_19ostreambuf_iteratorIcNS_11char_traitsIcEEEEEE\x00NSt3__211__money_putIcEE\x00NSt3__29money_putIwNS_19ostreambuf_iteratorIwNS_11char_traitsIwEEEEEE\x00NSt3__211__money_putIwEE\x00NSt3__28messagesIcEE\x00NSt3__213messages_baseE\x00NSt3__217__widen_from_utf8ILj32EEE\x00NSt3__27codecvtIDic11__mbstate_tEE\x00NSt3__212codecvt_baseE\x00NSt3__216__narrow_to_utf8ILj32EEE\x00NSt3__28messagesIwEE\x00NSt3__27codecvtIcc11__mbstate_tEE\x00NSt3__27codecvtIwc11__mbstate_tEE\x00NSt3__27codecvtIDsc11__mbstate_tEE\x00NSt3__26locale5__impE\x00NSt3__25ctypeIcEE\x00NSt3__210ctype_baseE\x00NSt3__25ctypeIwEE\x00false\x00true\x00NSt3__28numpunctIcEE\x00NSt3__28numpunctIwEE\x00vector\x00NSt3__214__shared_countE\x00basic_string\x00NSt3__214error_categoryE\x00NSt3__212system_errorE\x00NSt3__212__do_messageE\x00: \x00terminating with %s exception of type %s: %s\x00terminating with %s exception of type %s\x00terminating with %s foreign exception\x00terminating\x00uncaught\x00St9exception\x00N10__cxxabiv116__shim_type_infoE\x00St9type_info\x00N10__cxxabiv120__si_class_type_infoE\x00N10__cxxabiv117__class_type_infoE\x00pthread_once failure in __cxa_get_globals_fast()\x00cannot create pthread key for __cxa_get_globals()\x00cannot zero out thread value for __cxa_get_globals()\x00terminate_handler unexpectedly returned\x00terminate_handler unexpectedly threw an exception\x00std::bad_alloc\x00St9bad_alloc\x00St11logic_error\x00St13runtime_error\x00St12length_error\x00std::bad_cast\x00St8bad_cast\x00N10__cxxabiv119__pointer_type_infoE\x00N10__cxxabiv117__pbase_type_infoE\x00N10__cxxabiv121__vmi_class_type_infoE'
```

Comment:
Thank you for addressing my previous comments! Unfortunately, there is another fat array a few lines below that I'd also like to be truncated. I should have mentioned that explicitly, sorry for that.

Reply:
Don't worry. I've changed it :)